src/utils/opt_in_checker.py

```diff
@@ -0,0 +1,255 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import time
+from enum import Enum
+from pathlib import Path
+from platform import system
+from sys import stdin
+
+from .colored_print import colored_print
+from .input_with_timeout import input_with_timeout
+
+
+class ISIPCheckResult(Enum):
+    DECLINED = 0
+    ACCEPTED = 1
+    NO_FILE = 2
+
+
+class DialogResult(Enum):
+    DECLINED = 0
+    ACCEPTED = 1
+    TIMEOUT_REACHED = 2
+
+
+class OptInChecker:
+    dialog_timeout = 50  # seconds
+    path_to_opt_in_out_script = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+    opt_in_out_script_name = "opt_in_out"
+    doc_link = "http://openvino-docs.inn.intel.com/"
```

Comment:
strange internal link. Why is it needed?

Reply:
We need a link to Openvino documentation in opt_in message.